pkg/apis/options/options.go

```diff
@@ -86,6 +86,7 @@ type Options struct {
 	OIDCJwksURL                        string   `flag:"oidc-jwks-url" cfg:"oidc_jwks_url"`
 	OIDCEmailClaim                     string   `flag:"oidc-email-claim" cfg:"oidc_email_claim"`
 	OIDCGroupsClaim                    string   `flag:"oidc-groups-claim" cfg:"oidc_groups_claim"`
+	OIDCClaimsParameter                string   `flag:"oidc-claims-parameter" cfg:"oidc-claims-parameter"`
```

Comment:
This line should be changed so that using this option within a config file or in combination with environment variables works as expected (here underscores are used instead of hyphens).

```suggestion
	OIDCClaimsParameter                string   `flag:"oidc-claims-parameter" cfg:"oidc_claims_parameter"`
```